lib/web_ui/lib/src/engine/text/font_collection.dart

```diff
@@ -130,7 +130,12 @@ class HtmlFontCollection implements FlutterFontCollection {
     }
 
     try {
-      fontFaces.forEach(domDocument.fonts!.add);
+      // Since we can't use tear-offs for interop members, this code is faster
+      // and easier to read with a for loop instead of forEach.
+      // ignore: prefer_foreach
```

Comment:
To avoid this lint override, you could instead do:
```
fontFaces.forEach((DomFontFace fontFace) => domDocument.fonts!.add(fontFace));
```

Reply:
This comes across another lint: https://dart.dev/tools/linter-rules#avoid_function_literals_in_foreach_calls :/

Reply:
Blech. Well then just ignore my comment for now. If we will eventually get tearoffs for static interop (you could even just have a kernel transform emit a closure like this for static interop tearoffs) maybe we should put a TODO here so that we can remove the lint override?

Reply:
Well I read your more detailed comment, thanks for the context. I was imagining dynamic tearoff creation but I didn't think about tearoff equality... hmmm.

Reply:
In general these lint rules are awfully opinionated about things that don't seem to matter all that much 😆

Reply:
Yeah, even if we were to ignore tearoff equality, the semantics is inconsistent between a regular call and a tear-off call, so in cases where optionality matters, users will still have to battle these lints.

I agree that there shouldn't be a lint for a regular for-loop, but that's my personal opinion. :)